tardis/tests/tests_slow/conftest.py

```diff
@@ -50,16 +49,13 @@ def plot_object(request):
     return PlotUploader(request)
 
 
-def setup_names():
-    slow_tests_dir = os.path.dirname(os.path.realpath(__file__))
-    names = list()
-    for child in os.listdir(slow_tests_dir):
-        if os.path.isdir(os.path.join(slow_tests_dir, child)):
-            names.append(child)
-    return names
+setup_names = [
+    name for name in os.listdir(os.path.dirname(os.path.realpath(__file__)))
+    if os.path.isdir(os.path.join(os.path.dirname(os.path.realpath(__file__)), name))
+]
```

Comment:
It is required to have only "names" of setup directories and not complete paths, so that we may use them for `reference_dirpath` and `setup_name` without `.split("/")[-1]`. Hence `os.path.listdir` is used instead of `glob.glob`.